runtime/src/accounts_index.rs

```diff
@@ -12,7 +12,7 @@ type AccountMapEntry<T> = (AtomicU64, RwLock<SlotList<T>>);
 
 #[derive(Debug, Default)]
 pub struct AccountsIndex<T> {
-    pub account_maps: HashMap<Pubkey, AccountMapEntry<T>>,
+    pub account_maps: BTreeMap<Pubkey, AccountMapEntry<T>>,
```

Comment:
I'm surprised only this change is needed for switching to `BTreeMap`.

Reply:
Any idea how this affects banking performance?

Reply:
I'll check that now. :)

Reply:
@sakridge Sorry for waiting you about this: I've prepared finally!:

- https://github.com/solana-labs/solana/pull/9527#issuecomment-627210347
- https://github.com/solana-labs/solana/pull/9527#issuecomment-627265007

Hope these clarifies any of your concerns! Feel free to ask any question too. :)

Reply:
Ok, cool. That's not too bad. I also ran banking-bench with this PR. On the machine I tested on, this PR brings the performance from 400k tps to 390k, so ~2%.